ci/Jenkinsfile

```
@@ -9,6 +9,13 @@ def wait_for_db() {
     }
 }
 
+def wait_for_pr_db() {
+    retry(6) {
+        sleep(time: 5, unit: 'SECONDS')
+        sh 'curl https://pr.permaplant.net/api/config'
```

To also fail unsuccessful responses from the server (5xx or similar):

```suggestion
        sh 'curl --fail https://pr.permaplant.net/api/config'
```

---

Perfect, thank you.